src/main/proto/cosmas.proto

```diff
@@ -37,9 +37,34 @@ message CreateVersionRequest {
 message CreateVersionResponse {
 }
 
+message CreateHistoryOfVersionRequest {
+    string user = 1;
+    string fileId = 2;
+    string patch = 3;
+    int64 timeStamp = 4;
+}
+
+message CreateHistoryOfVersionResponse {
+}
+
+message GetHistoryOfVersionRequest {
+    string fileId = 1;
+    int32 version = 2;

```

Comment:
What's the meaning of this "version" field?

Reply:
patches add in some order, version - number in this order

Reply:
Thanks. I don't think that thsi field is needed, because of the usage pattern, but anyway, if it is required for whatever reason, then let's name it appropriately. E.g. `patchNum` is okay. The matter is that `version` is already used in other request (and for better purpose) and if one sees two fields named "version" in different sorts of requests, it immediately raises the question "is it the same thing?". And the answer "no" is surprising.